client_sdk/go/fpc/management.go

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
might be worth to mention also in `docs/design/fabric-v2+/fpc-management.md` that there is a go management api (maybe just by a link to this go file?)

Reply:
good point! I added a TODO for the moment; once the attestation Credentials are correctly used I can update this.